### DISCUSSION TITLE
Browsing

### DISCUSSION THREAD

**Discussion by dilab** [manuelkiessling/php-ai-tool-bridge#5] [Ideas]
Cool idea. Can we use this to implement web browsing for a chat?

**Comment by manuelkiessling** [comment]
Well, the "bridge" part is completely up to you, in the sense that you can feed any kind of information into the AI conversation — it doesn't have to be the result of a database query or the content of an API request.

It would be totally feasible to write a bridge tool that scrapes the text content from a web page.

**Comment by dilab** [comment]
@manuelkiessling does it work for certain model (GPT-4) only?

**Comment by manuelkiessling** [comment]
@dilab It has been *verified to work* with the OpenAI gpt-4 model, but that doesn't necessarily mean that it wouldn't work with other models that know how to "behave" and follow instructions with enough precision.
